tests/echo/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,184 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`echo.php 1`] = `
+<?php
+
+echo "Hello World";
+
+echo "This spans
+multiple lines. The newlines will be
+output as well";
```

Comment:
(not the topic of this PR but) the line feeds inside this string are transformed into `\n`

this code:

```php
echo "This spans
multiple lines. The newlines will be
output as well";
```

becomes:

```php
echo "This spans\\nmultiple lines. The newlines will be\\noutput as well";
```

is this what we want?

Reply:
@nicoder it is bug in parser, will be fixed in new version :+1: When we update php parser we will update all snapshots :+1: